### ISSUE TITLE
HLS content: video is not playing.

### ISSUE THREAD

**Issue by Lewis945** [google/shaka-player#1057]
**Have you read the [FAQ](https://goo.gl/JE1Sy5) and checked for duplicate issues**: yes

**What version of Shaka Player are you using**: latest

**Can you reproduce the issue with our latest release version**: yes

**Can you reproduce the issue with the latest code from `master`**: yes

**Are you using the demo app or your own custom app**: demo

**If custom app, can you reproduce the issue using our demo app**: - 

**What browser and OS are you using**: Chrome or Edge, Windows 10

**What are the manifest and license server URIs**: attached zip archive with content for this issue
*(you can send the URIs to <shaka-player-issues@google.com> instead, but please use GitHub and the template for the rest)*

**What did you do?** tried playing hls content

**What did you expect to happen?** video and audio to be played

**What actually happened?** only audio was played

Can someone validate m3u8 manifest files? Is this an issue of shaka player or manifest files? 
HLS.js player played this content successfully.
HLS content was created manually from mp4box dash manifest due to specification.

**Comment by vaage** [comment]
@Lewis945 I have looked through the content you sent us and I have found what I believe to be the issue. It appears that your last media playlist is setting audio and video to audio. If I remove the entries for Manifest6 I can get video. I also looked at what Streaming Engine was trying to stream and both were video and audio streams were actually audio.

Behind the scenes Shaka Player sees that you have an entry for video and audio, and discards all the video-only entires. It then goes to play that variant and since it only has audio codecs, it plays the audio.
